index.js

```diff
@@ -22,6 +22,8 @@ module.exports = {
     // bundle scripts from vendor folder
     this.import('vendor/@esri/arcgis-rest-request/request.umd.js');
     this.import('vendor/@esri/arcgis-rest-auth/auth.umd.js');
+    this.import('vendor/shims/@esri/arcgis-rest-request.js');
```

Comment:
I think the reason the above `import`s don't work is b/c these shim files don't exist. I'm surprised that doesn't result in a build error. You can (and should) copy the relevant ones from https://github.com/Esri/ember-arcgis-portal-services/tree/master/vendor/shims/%40esri

Reply:
i think the shims were working the whole time and i was just running into an unrelated error with the app's cache.